src/main/java/org/embulk/filter/column/ColumnFilterPlugin.java

```diff
@@ -117,7 +120,7 @@ static Schema buildOutputSchema(PluginTask task, Schema inputSchema)
                 boolean matched = false;
                 for (ColumnConfig dropColumn : dropColumns) {
                     // skip json path notation to build outputSchema
-                    if (dropColumn.getName().startsWith("$.")) {
+                    if (PathCompiler.isDocContext(dropColumn.getName())) {
```

Comment:
Why was the method name changed to "isDocContext"?
Is it the original name in jsonpath library?

I could not infer what is doc context, so I may want an alias for the method and use it.